contracts/metatx/ERC2771Context.sol

```diff
@@ -1,15 +1,15 @@
 // SPDX-License-Identifier: MIT
 // OpenZeppelin Contracts v4.3.2 (metatx/ERC2771Context.sol)
 
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.9;
 
 import "../utils/Context.sol";
 
 /**
  * @dev Context variant with ERC2771 support.
  */
 abstract contract ERC2771Context is Context {
-    address private _trustedForwarder;
+    address private immutable _trustedForwarder;
```

Comment:
Do we want this to be immutable in Upgradeable Contracts?

Reply:
I think so.